charts/ingress-nginx/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 2.4.0
+version: 2.5.0
```

```suggestion
version: 2.6.0
```

Another change got in before yours. Please bump the chart version and squash the commit. After that this lgtm

---

done

---

Sorry about this. Yet another change was merged in before yours again. Gonna need to bump the minor version and rebase your changes.

---

do you mean chart version 2.6.1 ?

---

So we should bump to `2.7.0`

Kubernetes follows [semver](https://semver.org/), [k8s docs](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions). So Major.Minor.Patch

---

gotcha, done